config_defaults.yml

```diff
@@ -23,6 +23,9 @@ Gemfile:
       - gem: puppetlabs_spec_helper
         version: '~>0.10.3'
       - gem: puppet_facts
+      # newer version required for ruby 1.9 compat
+      - gem: puppet-blacksmith
+        version: '>= 3.4.0'
```

Comment:
Take a look at https://github.com/puppetlabs/puppetlabs-reboot/blob/00d4535df025b17d250f89229fd34fbb7f313361/.sync.yml#L13 for the additional bit for ensuring just on non-Windows platforms.